bftengine/src/bftengine/MessageBase.cpp

```diff
@@ -119,5 +119,76 @@ namespace bftEngine
 			msgSize_ = size;
 		}
 
+		MessageBase* MessageBase::cloneObjAndMsg() const
```

Comment:
I know this is not possible in this PR but we should rework the code so we aren't passing around raw pointers for messages.